addons/website/static/src/js/widgets/theme.js

```diff
@@ -541,6 +547,21 @@ var ThemeCustomizeDialog = Dialog.extend({
 
         return def;
     },
+    /**
+     * @private
+     */
+    _setAuto: function ($inputData) {
+        var self = this;
+        var value = $inputData.data('value');
```

Comment:
You don't need that extra variable as it is used once and the expression is small.